lua/class_definitions.lua

```diff
@@ -75,6 +75,24 @@ classes = {
         functions = {
         }
     },
+    calendar = {
+        attributes = {
+        },
+        functions = {
+            { name = "get_turn", rval = "int", args = { } },
+            { name = "minutes_past_midnight", rval = "int", args = { } },
+            { name = "seconds_past_midnight", rval = "int", args = { } },
+            { name = "sunrise", rval = "calendar", args = { } },
+            { name = "sunset", rval = "calendar", args = { } },
+            { name = "is_night", rval = "bool", args = { } },
+            { name = "sunlight", rval = "calendar", args = { } },
```

Comment:
How would I go about adding this? I was looking through the generate_binding code, but I'm not entirely sure what needs to be done.